.github/workflows/knative-go-build.yaml

```diff
@@ -42,8 +43,17 @@ jobs:
       - name: Check out code
         uses: actions/checkout@v2
 
-      - name: Vet
-        run: go vet ./...
-
       - name: Build
-        run: go build -v ./...
+        run: |
+          tags="$(grep -I  -r '// +build' . | \
+                grep -v '^./vendor/' | \
+                grep -v '^./hack/' | \
+                grep -v '^./third_party' | \
+                cut -f3 -d' ' | \
+                sort | uniq | \
+                grep -v '^!' | \
+                tr '\n' ' ')"
+
+          echo "Building with tags: ${tags}"
+          go test -vet=off -tags "${tags}" -run=^$ ./... | grep -v "no test" || true
```

Comment:
This looks custom, if you want to preserve this, you should rename this file or the knobots will attempt to clobber it ~daily.

/hold

Reply:
looking at the wrong side 🤣